grakn-migration/csv/src/main/java/ai/grakn/migration/csv/Main.java

```diff
@@ -40,12 +39,15 @@
  */
 public class Main {
 
+    static boolean newClusterManager = false;
+
     public static void main(String[] args) {
         start(null, args);
     }
 
     public static void start(ClusterManager manager, String[] args){
```

Comment:
Something smells bad here if a migrator needs to start an Engine ClusterManager. Why is that?